src/graphql/account.rs

```diff
@@ -301,7 +301,25 @@ impl AccountMutation {
         Ok(username)
     }
 
-    /// Authenticates with the given username and password
+    /// Authenticates with the given username and password.
+    ///
+    /// If the `lastSigninTime` value of the `account` is `None`, the operation will fail, and
+    /// it should be guided to call `signInWithNewPassword` GraphQL API.
+    ///
+    /// If the `signIn` GraphQL API response is the error message "a password change is
+    /// required to proceed", you can call the `signInWithNewPassword` GraphQL API.
+    ///
+    /// ```ignore
+    /// r#"mutation {
+    ///     signInWithNewPassword(username: "user", password: "pw", newPassword: "pw2") {
+    ///         token
+    ///     }
+    /// }"#
+    /// ```
+    ///
+    /// # Errors
+    ///
+    /// An error message (`String`) if any of the checks fail or an error occurs during the process.
```

Comment:
I suggest:

```rust
/// # Errors
///
/// Returns `Err` if the password is invalid, this is the first sign-in attempt, the access doesn't originate from a permitted IP address, or the number of sessions exceeds the maximum limit.
```

The style where the sentence starts with a third-person singular verb is a convention in Rust documentation. Additionally, it would be better to specify exactly the cases where errors occur.

Reply:
I'll change it.